samples/apps-in-meeting/csharp/Content Bubble/Bots/ContentBubbleBot.cs

```diff
@@ -0,0 +1,192 @@
+
+                    var path = "./Cards/AdaptiveThirdTitle.json";
+                    var adaptiveCardJson = File.ReadAllText(path);
+                    var adaptiveCardAttachment = new Attachment
+                    {
+                        ContentType = AdaptiveCard.ContentType,
+                        Content = JsonConvert.DeserializeObject(adaptiveCardJson),
+                    };
+
+                    await turnContext.SendActivityAsync(MessageFactory.Attachment(adaptiveCardAttachment));
+                }
+
+                else if (selectedoption == "yes")
+                {
+                    await turnContext.SendActivityAsync(turnContext.Activity.From.Name + " : " + "**Yes** for " + "'" + command + "'");
+                }
+
+                else if (selectedoption == "no")
+                {
+                    await turnContext.SendActivityAsync(turnContext.Activity.From.Name + " : " + "**No** for " + "'" + command + "'");
+                }
+            }
+        }
+        protected override async Task OnMembersAddedAsync(IList<ChannelAccount> membersAdded, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
+        {
+            var welcomeText = "Hello and welcome!";
+            foreach (var member in membersAdded)
+            {
+                if (member.Id != turnContext.Activity.Recipient.Id)
+                {
+                    await turnContext.SendActivityAsync(MessageFactory.Text(welcomeText, welcomeText), cancellationToken);
+                }
+            }
+        }
+
+        protected override async Task<TaskModuleResponse> OnTeamsTaskModuleSubmitAsync(ITurnContext<IInvokeActivity> turnContext, TaskModuleRequest taskModuleRequest, CancellationToken cancellationToken)
+        {
+
+            var replyJson = JObject.Parse(taskModuleRequest.Data.ToString());
+            var reply = Convert.ToString(replyJson["userValue"]);
+            var title = Convert.ToString(replyJson["title"]);
+            if (reply == "yes")
+            {
+
+                await turnContext.SendActivityAsync(turnContext.Activity.From.Name + " : " + "**Yes** for " + "'" + title + "'");
+            }
+            else
+            {
+                await turnContext.SendActivityAsync(turnContext.Activity.From.Name + " : " + "**No** for " + "'" + title + "'");
+
+            }
+            return null;
+        }
+
+        public async static Task FetchTaskFirst(ITurnContext turnContext)
+        {
+            Activity activity = MessageFactory.Text("**Please provide your valuable feedback**");
+
+            activity.ChannelData = new TeamsChannelData
+            {
+                Notification = new NotificationInfo()
+                {
+                    AlertInMeeting = true,
+
+                    ExternalResourceUrl = "https://teams.microsoft.com/l/bubble/<<APP-ID>>?url=<<BASE-URL>>/FirstPage&height=<HEIGHT>&width=<WIDTH>&title=ContentBubble&completionBotId=<<BOT-ID>>"
```

Comment:
Please fill in the  AppId, Based URL from config file. Also, set fixed height and width.

Reply:
Done